services/core-web/src/components/Forms/incidents/IncidentForm.js

```diff
@@ -942,7 +942,7 @@ const renderEditSaveControls = (childProps, isEditMode, isNewIncident) => (
         <Popconfirm
           placement="topLeft"
           title="Are you sure you want to cancel this submission? All unsaved changes will be lost."
-          onConfirm={() => childProps.handlers.handleCancelEdit(isNewIncident)}
```

Comment:
eslint was angry that the original definition of this from MineIncident was reusing a variable name already declared in upper scope, and I thought that we didn't need a parameter at all.